lib/task_helpers/metrics.rb

```diff
@@ -0,0 +1,42 @@
+module TaskHelpers
+  class Metrics
+    def self.per_tenant
+      data = [%w(tenant_id external_tenant_id number_of_portfolios number_of_products
+                number_of_portfolios_shared number_of_groups_per_portfolios_shared number_of_orders_per_day)]
```

Comment:
For readability we should list each column on a separate line and it matches with what you are doing below in the `data.push` section.